core-web/nx.json

```diff
@@ -106,6 +106,16 @@
         "@nx/eslint:lint": {
             "inputs": ["default", "{workspaceRoot}/.eslintrc.json"],
             "cache": true
+        },
+        "@nx/rollup:rollup": {
+            "cache": true,
+            "dependsOn": ["^build"],
+            "inputs": ["production", "^production"]
+        },
```

Comment:
Do we need to remove this?

Reply:
I will remove it when I migrate the other libs.

Reply:
And when is that?